css/style.css

```diff
@@ -5,3 +5,27 @@ html {
 }
 
 /*put all other styles below this line, please*/
+
+body {
+  display: grid;
+  grid-template-rows: min-content;
+  grid-template-rows: max-content;
```

Comment:
hint: `grid-template-rows: min-content min-content max-content min-content;`

(it's a shorthand syntax)